src/openzeppelin/account/account.cairo

```diff
@@ -46,6 +46,23 @@ mod Account {
         public_key: felt252
     }
 
+    #[event]
+    #[derive(Drop, starknet::Event)]
+    enum Event {
+        OwnerAdded: OwnerAdded,
+        OwnerRemoved: OwnerRemoved,
+    }
+
+    #[derive(Drop, starknet::Event)]
+    struct OwnerAdded {
+        new_owner_key: felt252
```

Comment:
In the discussed spec `new_owner_guid` is used. GUID stands for "global unique id" as I've been told. With this said, I wouldn't say I like the new_owner_guid term that much, but I think we need to keep consistency with the rest of the accountors.

Reply:
Haha yeah, GUID was alien to me. I'm used to seeing UUID. And I agree, I don't love it hence the change.

If we want consistency here, I'd push for standardizing this. AFAIK we don't have that. The only thing I could find is this discussion from last year: https://community.starknet.io/t/account-keys-and-addresses-derivation-standard/1230

If you agree, I'm thinking this should be discussed in SNIP-6. We haven't been including events in the interface and maybe this is a good place to start

Reply:
@andrew-fleming i don't think these events should be part of SNIP-6 since they're not for any kind of account (as SNIP-6 aims to be) but for key-controlled ones